Assets/Scripts/UI/PauseMenu.cs

```diff
@@ -72,4 +83,55 @@ public void QuitGame()
         Debug.Log("Quitting game...");
         Application.Quit();
     }
-}
+
+    public void OpenSoundMenu()
+    {
+        soundMenuUI.SetActive(true);
+        pauseMenuUI.SetActive(false);
+    }
+
+    public void ReturnToPauseMenu()
+    {
+        if (soundMenuUI.activeInHierarchy)
+        {
+            soundMenuUI.SetActive(false);
+            pauseMenuUI.SetActive(true);
+        }
+        else if (controlMenuUI.activeInHierarchy)
+        {
+            controlMenuUI.SetActive(false);
+            pauseMenuUI.SetActive(true);
+        }
+    }
+
+    public void ExitfromSoundSetting()
+    {
+        soundMenuUI.SetActive(false);
+        if (!loginPanelUI.activeInHierarchy && !controlPanelUI.activeInHierarchy)
+        {
+            HUDEnableEvent.TriggerEvent();
+            Time.timeScale = 1f;
+            Cursor.lockState = CursorLockMode.Locked;
+            Cursor.visible = false;
```

Comment:
Can probably be made into a new function so that it can be reused in `ExitfromControlMenu`.

Reply:
[3872a39](https://github.com/ectodrop/loop-game/pull/112/commits/3872a3979d75229190167d87512711d71e7aab0b)